unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/VisualTests/WearablesVisualTests.cs

```diff
@@ -87,23 +93,40 @@ private IEnumerator LoadWearable(string wearableId, string bodyShapeId, Vector3
     {
         catalog.TryGetValue(wearableId, out WearableItem wearableItem);
         Assert.NotNull(wearableItem);
+
         WearableController wearable = new WearableController(wearableItem);
         toCleanUp.Add(wearable);
+
         bool succeeded = false;
         bool failed = false;
-        wearable.Load(bodyShapeId, CreateTestGameObject(wearable.id, wearablePosition).transform, x => succeeded = true, x => failed = true);
+
+        GameObject testGameObject = CreateTestGameObject(wearable.id, Vector3.up * -0.75f);
+        wearable.Load(bodyShapeId, testGameObject.transform, x => succeeded = true, x => failed = true);
+
         yield return new WaitUntil(() => succeeded || failed);
+
         Assert.IsTrue(succeeded);
+
         wearable.SetAssetRenderersEnabled(true);
-        wearable.SetupDefaultMaterial(avatarMaterial, skinColor, hairColor);
+        wearable.SetupHairAndSkinColors(skinColor, hairColor);
+
+        var rends = wearable.GetRenderers();
+        combiner.Combine(rends[0], rends.ToArray(), avatarMaterial);
```

Comment:
Now, the combiner acts on the wearable tests too, so the tests reflect the current state of this PR.